drivers/hmis/app/models/hmis/form/instance_project_match.rb

```diff
@@ -0,0 +1,65 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# determine if project is a valid match for a form instance. The match is ranked so that when there are multiple form
+# definitions that match a project, we can take the definition with the best rank
+#
+class Hmis::Form::InstanceProjectMatch
+  include Memery
+  attr_accessor :project, :instance
+
+  # match types ordered by rank. Lower rank is better
```

Comment:
```suggestion
  # Match types ordered by rank, from most specific to least specific. lower rank number if more specific.
```
maybe? "Better" was unclear to me